cinema/permissions.py

```diff
@@ -0,0 +1,12 @@
+from rest_framework.permissions import BasePermission, SAFE_METHODS
```

Comment:
There is a syntax error in the import statement. The correct syntax should be `from rest_framework.permissions import BasePermission, SAFE_METHODS`. Ensure there is no trailing slash or incorrect character.

Reply:
I have exactly the same line